tutorials/run-symfony-on-appengine-flexible/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Run Symfony on Google App Engine Flexible Environment
+description: Learn how to deploy a Symfony app to Google App Engine flexible environment.
+author: bshaffer
+tags: App Engine, Symfony, PHP
+date_published: 2017-03-15
+---
+## Symfony
+
+> [Symfony][symfony] is a set of PHP Components, a Web Application framework, a
+> Philosophy, and a Community — all working together in harmony.
+>
+> – symfony.com
+
+You can check out [PHP on Google Cloud Platform][php-gcp] to get an
+overview of PHP itself and learn ways to run PHP apps on Google Cloud
+Platform.
+
+## Prerequisites
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
```

Comment:
I copied this over from [other community tutorials](https://github.com/bshaffer/community/blob/51ce1e543dcbd199327c8b12539f9974832f1588/tutorials/run-geddyjs-on-google-app-engine.md), and our [official docs](https://cloud.google.com/resource-manager/docs/creating-managing-projects) are hardly more clear. I could just link to the official docs instead of directly to the console, WDYT?